script/jenkins/test.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+source ./_beats/dev-tools/common.bash
+
+jenkins_setup
+
+#cleanup() {
+#  rm -rf $TEMP_PYTHON_ENV
+#  make stop-environment fix-permissions
+#}
+#trap cleanup EXIT
+
+make testsuite
```

Comment:
This will also run the `unit-tests` again. How about you remove the `unit-test` step for now and call this step `linux-tests`?